File: Fantasy/src/cc/fantasy/swing/Justify.java
package cc.fantasy.swing;

import javax.swing.SwingConstants;

public enum Justify {

    TOP(SwingConstants.TOP),
    LEFT(SwingConstants.LEFT),
    RIGHT(SwingConstants.RIGHT),
    BOTTOM(SwingConstants.BOTTOM),
    CENTER(SwingConstants.CENTER);
    
    private final int justify;
    
    private Justify(int j) {
        justify = j;
    }
    
    int getJustify() {
        return justify;
    }
    
}

File: GameLibrary/src/cc/lib/game/Justify.kt
package cc.lib.game

enum class Justify {
	LEFT,
	RIGHT,
	TOP,
	BOTTOM,
	CENTER
	;

	companion object {
		open fun verticalEntries(): Array<Justify> = arrayOf(TOP, BOTTOM, CENTER)
		open fun horizontalEntries(): Array<Justify> = arrayOf(LEFT, RIGHT, CENTER)
	}
}
